src/mod2c_core/nocpout.c

```diff
@@ -2302,6 +2355,7 @@ Sprintf(buf, " _ion_%s = %s;\n", SYM(q1)->name, SYM(q1)->name);
 			Lappendstr(l, buf);
 			Sprintf(buf, "    Memb_list* _%s_ml;\n", in);
 			Lappendstr(l, buf);
+			// TODO: here _type is a static global variable even though warning is not shown. Fix this.
 			Sprintf(buf, "    _%s_ml = _nt->_ml_list[_%s_type];\n", in, in);
```

Comment:
note to myself: this is also covered by channel-benchmark https://bbpgitlab.epfl.ch/hpc/channel-benchmark/-/blob/master/benchmark/channels/lib/modlib/CaDynamics_E2.mod